src/vma/dev/ib_ctx_handler_collection.cpp

```diff
@@ -117,6 +117,25 @@ ib_ctx_handler* ib_ctx_handler_collection::get_ib_ctx(struct ibv_context* p_ibv_
 	return NULL;
 }
 
+ib_ctx_handler* ib_ctx_handler_collection::get_ib_ctx(const char *ifa_name)
+{
+	int fd;
+	char ib_path[IBV_SYSFS_PATH_MAX];
+
+	ib_context_map_t::iterator ib_ctx_iter;
+	for (ib_ctx_iter = m_ib_ctx_map.begin(); ib_ctx_iter != m_ib_ctx_map.end(); ib_ctx_iter++) {
+		snprintf(ib_path, sizeof(ib_path), "/sys/class/infiniband/%s/device/net/%s/ifindex", ib_ctx_iter->first->device->name, ifa_name);
```

Comment:
Can you add result verification for `snprintf` to pass security check.
Example: https://github.com/Mellanox/libvma/blob/master/src/vma/util/utils.h#L300
Key steps:
- char ib_path[IBV_SYSFS_PATH_MAX]={0};
- int n = -1; n=snprintf();
- if (likely((0 < n) && (n < (int)sizeof(ib_path))))